helm/sematic-grafana-dashboards/templates/sematic_runs_dashboard_configmap.yaml

```diff
@@ -0,0 +1,637 @@
+                "mode": "palette-classic"
+              },
+              "custom": {
+                "axisCenteredZero": false,
+                "axisColorMode": "text",
+                "axisLabel": "",
+                "axisPlacement": "auto",
+                "barAlignment": 0,
+                "drawStyle": "line",
+                "fillOpacity": 0,
+                "gradientMode": "none",
+                "hideFrom": {
+                  "legend": false,
+                  "tooltip": false,
+                  "viz": false
+                },
+                "lineInterpolation": "linear",
+                "lineWidth": 1,
+                "pointSize": 5,
+                "scaleDistribution": {
+                  "type": "linear"
+                },
+                "showPoints": "auto",
+                "spanNulls": false,
+                "stacking": {
+                  "group": "A",
+                  "mode": "none"
+                },
+                "thresholdsStyle": {
+                  "mode": "dashed"
+                }
+              },
+              "mappings": [],
+              "max": 1,
+              "thresholds": {
+                "mode": "percentage",
+                "steps": [
+                  {
+                    "color": "green",
+                    "value": null
+                  },
+                  {
+                    "color": "red",
+                    "value": 80
+                  }
+                ]
+              },
+              "unit": "percentunit"
+            },
+            "overrides": []
+          },
+          "gridPos": {
+            "h": 9,
+            "w": 24,
+            "x": 0,
+            "y": 14
+          },
+          "id": 5,
+          "options": {
+            "legend": {
+              "calcs": [],
+              "displayMode": "list",
+              "placement": "bottom",
+              "showLegend": true
+            },
+            "tooltip": {
+              "mode": "single",
+              "sort": "none"
+            }
+          },
+          "targets": [
+            {
+              "datasource": {
+                "type": "{{ .Values.datasources.prometheus.type }}",
+                "uid": "{{ .Values.datasources.prometheus.uid }}"
+              },
+              "editorMode": "builder",
+              "expr": "avg(container_memory_working_set_bytes{pod=~\".*sematic-driver.*$run_id.*|.*sematic-worker.*$run_id.*\", namespace=\"$namespace\"}) by (container_name, pod) / avg(cluster:namespace:pod_memory:active:kube_pod_container_resource_limits{pod=~\".*sematic-driver.*$run_id.*|.*sematic-worker.*$run_id.*\", namespace=\"$namespace\", resource=\"memory\"}) by (container_name, pod)",
+              "instant": true,
+              "interval": "",
+              "key": "Q-1d737459-4c0f-453b-8c26-201c3a996e5b-0",
+              "range": true,
+              "refId": "A"
+            }
+          ],
+          "title": "Memory usage fraction (namespace: $namespace)",
```

Comment:
This panel is erroring out for me on our deployment. Not sure if it's a bug?

Reply:
what error are you seeing here?  seems to work fine for me?

Reply:
i see some issues [1] reported in grafana that are very similar, all of which have been resolved as a bug in grafana itself, only pertaining to the editor.  since this does not affect graph rendering, i'm going to close this as won't fix, but can investigate further if it leads to any other issues.

[1] https://github.com/grafana/grafana/issues/44273

Reply:
sounds good!